ts_src/ethereum/index.ts

```diff
@@ -1,254 +1,4 @@
-        return gwei;
-    }
-
-    // Official JSON-RPC commands supported by Ethereum nodes
-    // Returns code at a given address.
-    async getCode(address: string, blockTagOrNumberOrHash?: string): Promise<string> {
-        let code = await this.provider.getCode(address, blockTagOrNumberOrHash);
-        return code;
-    }
-
-    // Official JSON-RPC commands supported by Ethereum nodes
-    // Returns a list of available compilers in the client.
-    // async getCompilers() {
-
-    // }
-
-    // Official JSON-RPC commands supported by Ethereum nodes
-    // Returns compiled code of a Solidity smart contract.
-    // async compileSolidity() {
-
-    // }
-
-    // Official JSON-RPC commands supported by Ethereum nodes
-    // Returns compiled code of an LLL program.
-    // async compileLLL() {
-
-    // }
-
-    // Official JSON-RPC commands supported by Ethereum nodes
-    // Returns compiled code of a Serpent program.
-    // async compileSerpent() {
-
-    // }
-}
-
-
-export enum Providers {
-    Alchemy,
-    Infura,
-    JsonRpc
-}
-
-export default EthClient;
-
-/* List of Ethereum JSON-RPC methods
-
-async blockNumber(): Returns the number of most recent block.
-async getBalance(): Returns the balance of the account of a given address.
-async getStorageAt(): Returns the value from a storage position at a given address.
-async getTransactionCount(): Returns the number of transactions sent from an address.
-async getBlockByHash(): Returns information about a block based on its hash.
-async getBlockByNumber(): Returns information about a block based on its number.
-async getTransactionByHash(): Returns the information about a transaction requested by transaction hash.
-async getTransactionByBlockHashAndIndex(): Returns information about a transaction by block hash and transaction index.
-async getTransactionByBlockNumberAndIndex(): Returns information about a transaction by block number and transaction index.
-async getTransactionReceipt(): Returns the receipt of a transaction by transaction hash.
-async getLogs(): Returns an array of all logs matching a given filter object.
-async sendTransaction(): Creates new message call transaction or a contract creation for signed transactions.
-async sendRawTransaction(): Sends a raw transaction.
-async call(): Executes a new message call immediately without creating a transaction on the blockchain.
-async estimateGas(): Generates and returns an estimate of how much gas is necessary to allow the transaction to complete.
-async getCode(): Returns code at a given address.
-async getCompilers(): Returns a list of available compilers in the client.
-async compileSolidity(): Returns compiled code of a Solidity smart contract.
-async compileLLL(): Returns compiled code of an LLL program.
-async compileSerpent(): Returns compiled code of a Serpent program.
-
-*/
+import { JsonRpcProvider, AlchemyProvider, InfuraProvider } from 'ethers';
+import { EthClient } from './eth-client';
```

Comment:
Not sure this is is the best place for those provider imports. Eventually we'd want to have them importing in eth-client. This is more a to-do that can come when we polish

Reply:
Added a commented TODO for later discussion